boards/common/atxmega/Makefile.include

```diff
@@ -0,0 +1,51 @@
+INCLUDES += -I$(RIOTBOARD)/common/atxmega/include
+
+PORT_LINUX ?= /dev/ttyACM0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
+
+SIZEFLAGS = --format=avr --mcu=$(CONFIG_CPU_MODEL)
```

Comment:
```suggestion
```

This breaks the default target for me. I get

```
avr-size: invalid argument to --format: avr
Usage: avr-size [option(s)] [file(s)]
 Displays the sizes of sections inside binary files
 If no input file(s) are specified, a.out is assumed
 The options are:
  -A|-B|-G  --format={sysv|berkeley|gnu}  Select output style (default is berkeley)
  -o|-d|-x  --radix={8|10|16}         Display numbers in octal, decimal or hex
  -t        --totals                  Display the total sizes (Berkeley only)
            --common                  Display total size for *COM* syms
            --target=<bfdname>        Set the binary file format
            @<file>                   Read options from <file>
  -h        --help                    Display this information
  -v        --version                 Display the program's version

avr-size: supported targets: elf32-avr elf32-little elf32-big srec symbolsrec verilog tekhex binary ihex plugin
make: *** [/home/maribu/Repos/software/RIOT/examples/default/../../Makefile.include:631: print-size] Error 1
```

The actual invocation is `avr-size --format=avr --mcu= /home/maribu/Repos/software/RIOT/examples/default/bin/atxmega-a1-xpro/default.elf`, so `CONFIG_CPU_MODEL` is unset at this point.

I'm using a binutils 2.35.1 build from unpatched upstream sources. So I think many other users will also be affected with not having the special format supported.

Also, I believe it is better to just print the size consistently for all boards. One of the main advantages of RIOT is IMO that one gets a consistent look and feel for all boards, and one can pick a different MCU by a different vendor for new project without having to worry that this has surprising impact on the development process.